src/api.rs

```diff
@@ -65,6 +65,9 @@ pub trait ElectrumApi {
         self.transaction_broadcast_raw(&buffer)
     }
 
+    /// Executes the requested API call returning the raw answer.
+    fn raw_call(&self, call: &Call) -> Result<serde_json::Value, Error>;
```

Comment:
this leads to calling method like `raw_call(&("method_name".to_owned(), vec![]))`, which is very non-rust ideomatic. Thus I propose a follow-up PR with changing function signature to `raw_call(&self, method_name: &str, params: impl IntoIterator<Item=Param>)`.

This, however, will be a breaking change and will require a new minor version bump.

Reply:
Opened https://github.com/bitcoindevkit/rust-electrum-client/pull/84 to implement your proposal